core/src/OpenACC/Kokkos_OpenACC_ParallelReduce_Range.hpp

```diff
@@ -42,10 +43,12 @@ class Kokkos::Impl::ParallelReduce<Functor, Kokkos::RangePolicy<Traits...>,
                                    ReducerType, Kokkos::Experimental::OpenACC> {
   using Policy = RangePolicy<Traits...>;
 
-  using Analysis = FunctorAnalysis<
-      FunctorPatternInterface::REDUCE, Policy,
-      std::conditional_t<std::is_same_v<InvalidType, ReducerType>, Functor,
-                         ReducerType>>;
+  using ReducerConditional =
+      Kokkos::Impl::if_c<std::is_same<InvalidType, ReducerType>::value, Functor,
+                         ReducerType>;
```

Comment:
```suggestion
  using ReducerConditional =
      Kokkos::Impl::if_c<std::is_same_v<InvalidType, ReducerType>, Functor,
                         ReducerType>;
```

Reply:
I amended the changes to Seyong's  commit